Lib/copy.py

```diff
@@ -84,22 +84,19 @@ def copy(x):
         return _copy_immutable(x)
 
     copier = getattr(cls, "__copy__", None)
-    if copier:
+    if copier is not None:
```

Comment:
Do you know that `if copier is not None` is slower than `if copier`?

But the change LGTM.

Reply:
I didn't know that. How much slower than `if foo`?